src/main/java/billgates/use_cases/alter_entry/AlterEntryUseCase.java

```diff
@@ -13,37 +23,60 @@ public AlterEntryUseCase(DatabaseGateway gateway) {
         this.gateway = gateway;
     }
 
-
+    /**
+     * This method aim to change numbers of column in one entry by once
+     * @param entryId   the ID of the entry which have a value we want to change.
+     * @param changeMap the map which has the alter column as key and new value as key
+     */
     @Override
-    public void alterEntry(int entryID, Object newValue, String alterColumn) {
-        int billID = User.getInstance().getCurrentBillID();
+    public void alterEntry(int entryId, Map<String, Object> changeMap) {
+        int billId = User.getInstance().getCurrentBillID();
 
-        QueryEntryData oldEntry = this.gateway.getEntryData(billID, entryID);
+        QueryEntryData oldEntry = this.gateway.getEntryData(billId, entryId);
         AlterEntryRequestModel model = new AlterEntryRequestModel(oldEntry);
```

Comment:
Although I don't think this is the proper use of a request model, it is okay for now.